src/clib/pio_darray.c

```diff
@@ -68,11 +71,12 @@ int PIOc_write_darray_multi(const int ncid, const int *vid, const int ioid,
                             bool flushtodisk)
 {
     iosystem_desc_t *ios;  /* Pointer to io system information. */
-    file_desc_t *file;
-    io_desc_t *iodesc;
-    int vsize, rlen;
-    int ierr = PIO_NOERR;
-    var_desc_t *vdesc0;
+    file_desc_t *file;     /* Pointer to file information. */
+    io_desc_t *iodesc;     /* Pointer to IO description information. */
+    int vsize;             /* ??? */
+    int rlen;              /* ??? */
+    var_desc_t *vdesc0;    /* ??? */
```

Comment:
vsize is the unit size in bytes of the given data.  rlen is the total data buffer size.